templates/all-services/all-services-en.html

```diff
@@ -0,0 +1,814 @@
+---
+{
+"title": "Government of Canada services",
+"language": "en",
+"altLangPage": "all-services-fr.html",
+"dateModified": "2024-11-21",
+"layout": "no-container",
+"css": ["https://use.fontawesome.com/releases/v5.8.1/css/all.css"]
+}
+---
+
+<!-- This page template is limited to the "All services" page on Canada.ca. For this reason, we are using utility classes instead of embedding the styles in this template. -->
+
+<div class="text-center"><a href="#all-topics" class="wb-sl">Skip to Access services by topic</a></div>
+
+<div class="container">
+	<h1 id="wb-cont">Government of Canada services</h1>
```

Comment:
Keep it left align, no need to center it.
And move it under the h1 because the master skip to content are going to bring the user on the h1 with the id wb-cont.

```suggestion

<div class="container">
	<h1 id="wb-cont">Government of Canada services</h1>
	<a href="#all-topics" class="wb-sl">Skip to Access services by topic</a>
```